ga4gh/datamodel/reads.py

```diff
@@ -197,7 +197,7 @@ class HtslibReadGroupSet(datamodel.PysamDatamodelMixin, AbstractReadGroupSet):
     Class representing a logical collection ReadGroups.
     """
     def __init__(
-            self, parentContainer, localId, samFilePath, backend):
+            self, parentContainer, localId, samFilePath, dataRepository):
```

Comment:
There's a couple of other places this will need to be changed. Doing a grep on 'backend' within ga4gh/datamodel will probably show them all.